templates/dbconfig.jndi.xml.epp

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<%# Some default are set in jira::config based on DB -%>
+<jira-database-config>
+  <name>defaultDS</name>
+  <delegator-name>default</delegator-name>
+  <database-type><%= $jira::config::dbtype %></database-type>
+<% if $jira::config::dbschema != undef { -%>
```

Comment:
```suggestion
<% if $jira::config::dbschema { -%>
```

Reply:
I used the dbconfig.xml.epp as an example and wanted to keep it similar.

Reply:
There are some cases where you want to explicitly check for undef, but yeah, in this case I don't think it's necessary since an empty string is not false.